src/group-constructors.js

```diff
@@ -22,9 +22,17 @@
     this._parent = null;
     this.children = children || [];
     this._reparent(this.children);
+    if (typeof timingInput == 'number') {
+      if (isNaN(timingInput)) {
+        timingInput = { duration: 0 };
+      } else {
+        timingInput = { duration: timingInput };
+      }
+    }
```

Comment:
This could be a function in shared scope, as you need to do it in multiple different places.